.github/workflows/code-coverage.yml

```diff
@@ -1,9 +1,5 @@
 name: Code coverage
 on:
-  push:
-    branches:
```

Comment:
It should be on master though.

Reply:
Will revert, was testing some things.

Reply:
Sure. Just as a reminder. Speaking of which, That show test results is a new build step/script. But I did not see the results, is that still work in progress, or did checked the wrong place?

Reply:
It is indeed still in progress. The problem with the method I am trying (waiting for the `Build` workflow to complete and then using the results in a new workflow) is that it only works when that workflow is on `master` (see https://github.com/orgs/community/discussions/25288). And since it isn't there, it doesn't show currently.